plugin/export_scripts_template/autoloads/godot_play_game_services.gd

```diff
@@ -25,12 +25,12 @@ func _ready() -> void:
 	
 	if not android_plugin:
 		if Engine.has_singleton(plugin_name):
-			print("Plugin found!")
+			print("GodotPlayGameServices plugin initialized successfully.")
 			
 			android_plugin = Engine.get_singleton(plugin_name)
 			android_plugin.initialize()
 		else:
-			printerr("No plugin found!")
+			printerr("Google Play Games Services are only supported on Android. The current platform does not support it.")
```

Comment:
I'm not really sure now, but I believe it's possible to reach this branch (i.e. the plugin is not found) even running on an android device, so in that case this error message can be confusing. What about "Google Play Games Services plugin not found".

Reply:
Godot Play Billing plugin uses similar message, so I thoght it will be nice to have the same here, but now I've checked their repo and they have one more branch. I'm ok with both solutions - anything that has the plugin name in it 😄 


Some context:
https://github.com/AchyutaStudios/godot-play-billing/blob/main/addons/play_billing/src/play_billing.gd#L134-L158

```gdscript
	if (OS.get_name() == "Android"):
		if (Engine.has_singleton("GodotPlayBilling")):
			_plugin = Engine.get_singleton("GodotPlayBilling")
			
			#...
			
			print("GodotPlayBilling plugin initialized successfully.")
		else:
			printerr("GodotPlayBilling not found. Make sure you have enabled 'Gradle Build' and the GodotPlayBilling plugin in your Android export settings! IAP will not work.")
	else:
		printerr("Google Play Billing is only supported on Android. The current platform does not support it.")

```